### PR TITLE
Add the ability to set empty-zones-enable

### PR DESCRIPTION
**Pull request by roidelapluie**
That option allows you to forward requests of PTR records of RFC1918
IP addresses.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -15,6 +15,7 @@
   $recursion            = $::dns::params::recursion,
   $allow_recursion      = $::dns::params::allow_recursion,
   $allow_query          = $::dns::params::allow_query,
+  $emptyzones_enable    = $::dns::params::empty_zones_enable,
```
Comment (ekohl) [inline manifests/init.pp]: ```emptyzones_enable``` is missing an underscore.

**Comment by roidelapluie** [comment]
Fixed and added test

**Comment by mmoll** [comment]
:+1:

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**